src/chapters/chapter_1/induction.tex

```diff
@@ -15,23 +14,12 @@ \subsection{Prove that $\sum_{i=1}^{n} i = n(n+1)/2$ for $n \geq 0$, by inductio
            &= 1
     \end{align}
 
-    Thus, the inductive step is true.
+    Thus, the base step is true.
 
     \textbf{Inductive step}
-    If all of $P(1) + ... + P(n)$ are true, then $P(n+1)$ is true. Let's try to
-    get the value of $P(n+1)$.
+    Assume all of $P(1), ..., P(n)$ are true. Then we show that $P(n+1)$ is true as well:
 
-    \begin{align}
-      P(1) + ... + P(n) + P(n+1) &= (n+1).((n+1)+1)/2\\
-      P(1) + ... + P(n) + P(n+1) &= (n+1).(n+2)/2\\
-      P(1) + ... + P(n) + P(n+1) &= (n^2 + 2n + 1n + 2)/2\\
-      P(1) + ... + P(n) + P(n+1) &= (n^2 + 3n + 2) / 2\\
-      n(n+1)/2 + (n+1) &= (n^2 + 3n + 2) / 2\\
-      (n^2 + 1n)/2 + (n+1) &= (n^2 + 3n + 2) / 2\\
-      (n^2 + 1n + 2(n+1))/2 &= (n^2 + 3n + 2) / 2\\
-      (n^2 + 1n + 2n + 2)/2 &= (n^2 + 3n + 2) / 2\\
-      (n^2 + 3n + 2)/2 &= (n^2 + 3n + 2) / 2\\
-    \end{align}
+    ...
```

Comment:
I leave redoing the inductive step without adding P(1)+...+P(n), which is invalid as discussed above, as an exercise for you.
You need to show that P(n+1) is true assuming P(1), ..., P(n) are true.
Start unfolding what P(n+1) means. What is P(n+1)?